uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/controllers/casetype.extended.controller.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-env amd */
+
+define(function () {
+  'use strict';
+
+  CaseTypeExtendedController.__name = 'CaseTypeExtendedController';
+  CaseTypeExtendedController.$inject = [
+    '$log', '$scope', 'crmApi'
+  ];
+
+  function CaseTypeExtendedController ($log, $scope, crmApi) {
+    $log.debug('Controller: CaseTypeExtendedController');
+
+    $scope.activityTypeOptions = [];
+
+    (function init () {
+      fetchActivityTypes();
+    })();
+
+    /**
+     * Fetch activity types which are components of CiviTask or CiviDocument
+     *
+     * @return {Promise}
+     */
+    function fetchActivityTypes () {
+      return crmApi('OptionValue', 'get', {
```

Comment:
if we decided to follow our own folders/files structure to keep things consistent and to disregard how things are done in core, then it should be the same for the `crmAPI` service. We don't use it anywhere in CiviHR, so i don't see why we shouldn't use our api/model/instance architecture

Reply:
to be clear i'm not asking you to write the correct implementation of `OptionGroup`, but the POC should show your intention to use it

Reply:
I added in the PR doc.